A32NX/ModelBehaviorDefs/A32NX/Interior/A32NX_Interior_ADIRS.xml

```diff
@@ -47,7 +47,7 @@
 
         <!-- Dummy button, using the FAULT variable -->
         <UseTemplate Name="FBW_Push_Toggle">
-            <TOGGLE_SIMVAR>L:A320_Neo_ADIRS_ALIGN_LIGHT_#ID#</TOGGLE_SIMVAR>
+            <TOGGLE_SIMVAR>L:A32NX_ADIRS_ALIGN_LIGHT_#ID#</TOGGLE_SIMVAR>
```

Comment:
You can remove the toggle simvar, since this is a dummy button, this will not have any use. This button cannot be pressed on the real A320 as well.